src/ashley/api/serializers.py

```diff
@@ -0,0 +1,13 @@
+"""Serializer for ashley api."""
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for User model."""
+
+    class Meta:
+        model = User
+        fields = ["id", "public_username"]
```

Comment:
You should also add info about its role. 
```suggestion
        fields = ["id", "public_username", "role"]
        
    role = serializers.SerializerMethodField()
    
    def get_role(self, obj):
        """Return the role associated to this user (student, moderator, instructor)."""
        // your code here
```